packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -1589,3 +1603,15 @@ cp1Rusd_sdeusdParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address
 cp1Rusd_sdeusdParentConfig_priceHaircutUnscaled = "0.075"
 cp1Rusd_sdeusdParentConfig_autoExchangeDiscountUnscaled = "0.005"
 cp1Rusd_sdeusdParentConfig_oracleNodeId = "<%= settings.sdeusdUsdcNodeIdStork %>"
+# -----------------------------------------------
+# TODO later: pass depositingEnabled through var (but there's no parseBool)
+# cp1Rusd_rusd_depositingEnabled = "true"
+cp1Rusd_rselini_capUnscaled = "0"
```

Comment:
can lift this one right before kicking off the rebalancing of the passive pool, can set to 2m instead of max uint

Reply:
we can raise this with deployment of oracle etc.

Reply:
ok